module/zfs/qat_compress.c

```diff
@@ -49,6 +49,14 @@ static Cpa16U num_inst = 0;
 static Cpa32U inst_num = 0;
 static boolean_t qat_dc_init_done = B_FALSE;
 int zfs_qat_compress_disable = 0;
+static Cpa32U
+adler32_swap(Cpa32U adler32)
+{
+	return (((adler32 & 0xff000000) >> 24) |
+	    ((adler32 & 0xff0000) >> 8) |
+	    ((adler32 & 0xff00) << 8) |
+	    ((adler32 & 0xff) << 24));
+}
```

Comment:
Can we use `BSWAP32()` for this?

Reply:
Is this macro `BSWAP_32()`?
https://github.com/zfsonlinux/zfs/blob/master/include/spl/sys/byteorder.h#L31-L34
Sure,  I will update this. Thanks for your suggestion.

Reply:
yes. that's the one.